dkg/dkg.go

```diff
@@ -313,6 +313,10 @@ func Run(ctx context.Context, conf Config) (err error) {
 
 	log.Info(ctx, "Successfully completed DKG ceremony 🎉")
 
+	// explicitly cancel the context and wait until the privkey lock is deleted
+	cancel()
```

Comment:
what if we call ctrl+C before getting to this line? This `cancel()` and `lockSvc.Done()` will not be called in that case. But defer cancel() at the start of this function will be called.

Reply:
even better, we can just defer `lockSvc.Done()` once since it'll be executed right before the exit, but after the `cancel()` is called at the end of the file